src/Host/Broker/Impl/Services/IExitService.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Threading;
+
+namespace Microsoft.R.Host.Broker.Services {
+    public interface IExitService {
+        void Exit();
```

Comment:
This is a temporary solution. I've created a task https://github.com/Microsoft/RTVS/issues/3411 to track refactoring of `CommonStartup`